.gitignore

```diff
@@ -2,3 +2,5 @@
 out
 node_modules
 package-lock.json
```

Comment:
```suggestion
```
I think `package-lock.json` should not be ignored. You can see below document with `npm help package-lock.json` command.

> This file is intended to be committed into source repositories,  and  serves  various  pur-
>       poses:
>
>       o Describe  a  single representation of a dependency tree such that teammates, deployments,
>         and continuous integration are guaranteed to install exactly the same dependencies.
>
>       o Provide a facility for users to "time-travel" to previous states of node_modules  without
>         having to commit the directory itself.
>
>       o To facilitate greater visibility of tree changes through readable source control diffs.
>
>       o And  optimize  the installation process by allowing npm to skip repeated metadata resolu-
>         tions for previously-installed packages.

Reply:
I'll revise this PR tomorrow :)